distributed_index_tests/lib/user.py

```diff
@@ -1,11 +1,188 @@
-import base64
+import requests
+import concurrent.futures
 import json
+import pprint
+import copy_reg
+import types
+import time
+import base64
+import document
+import uuid
+import re
+from collections import defaultdict
+
+# This function is added to use ProcessExecutor
+# concurrent.futures. 
+# 
+def _pickle_method(m):
```

Comment:
It looks like you are using only ThreadPools as executors. Do we need this anymore?